screens/AddMembers/AddMembers.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {View, Text, StyleSheet, Image, FlatList, ActivityIndicator } from 'react-native';
+import Contacts from '../Contacts/Contacts';
+import Background from '../../components/Background';
+import NHSStyle from '../../constants/NHSStyle';
+import Theme from '../../constants/Theme';
+import { SafeAreaView } from 'react-navigation';
+import { TextInput } from 'react-native-paper';
+
+
+export default class AddMembers extends React.Component {
+    constructor() {
+        super()
+        this.state={
+            isLoading: false,
+            contacts: []
+        }
+    }
+
+
+async componentDidMount() {
```

Comment:
We should better use useEffect() hook;

https://reactjs.org/docs/hooks-intro.html#motivation